Source/WebKit/WebKit.xcodeproj/project.pbxproj

```diff
@@ -3,7 +3,7 @@
 	archiveVersion = 1;
 	classes = {
 	};
-	objectVersion = 54;
+	objectVersion = 55;
```

Comment:
Is this intentional?

Reply:
I added the file through Xcode and it made these changes. I can remove them manually and see if it all still works.